docs/tutorials/g3-algebra-of-space.ipynb

```diff
@@ -861,7 +861,17 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.0"
+   "version": "3.7.3"
```

Comment:
Can you undo the changes to this file? I don't see the point in these, I think they're just cruft from an old / customized version of jupyter